client/src/main/java/org/apache/uniffle/client/impl/ShuffleWriteClientImpl.java

```diff
@@ -105,6 +106,7 @@ public class ShuffleWriteClientImpl implements ShuffleWriteClient {
   private final ExecutorService dataTransferPool;
   private final int unregisterThreadPoolSize;
   private final int unregisterRequestTimeSec;
+  private Set<ShuffleServerInfo> shuffleServerBlocklist;
```

Comment:
Why do we use the name `shuffleServerBlocklist`? What's the meaning of this variable?

Reply:
Blocklist is equal to Blacklist

Reply:
Block is a concept of our system. This name make me confusing.

Reply:
> Block is a concept of our system. This name make me confused.

I think so too, but i have not good idea. How about defectiveServerList? We need to unify our opinions. :joy:   @jerqi  @kaijchen  @advancedxy

Reply:
> I think so too, but i have not good idea. How about defectiveServerList? We need to unify our opinions. 😂 @jerqi @kaijchen @advancedxy

Maybe just `defectiveServers`? Because it's actually a `Set`.

Reply:
> > I think so too, but i have not good idea. How about defectiveServerList? We need to unify our opinions. 😂 @jerqi @kaijchen @advancedxy
> 
> Maybe just `defectiveServers`? Because it's actually a `Set`. And `withBlocklist` may be changed to `excludeDefectiveServers`.

It's ok for me.

Reply:
Done